Meta - Wearables for Android - Setup.md

```
@@ -0,0 +1,49 @@
+## Setting up your glasses
+
+1. Ensure your versions of the Meta AI app and glasses software are in line with the version dependencies [outlined here](https://wearables.developer.meta.com/docs/develop/dat/version-dependencies). Follow the instructions below to verify your current glasses version.
+2. Connect your glasses to the Meta AI app.
+3. Enable [developer mode](https://wearables.developer.meta.com/docs/develop/dat/getting-started-toolkit#enable-developer-mode-in-the-meta-ai-app) in the Meta AI app. Developer mode allows your unpublished app to register and interact with your AI glasses without the need to submit it for publishing review. Your app appears under **Meta AI settings** > **App connections** > **Developer mode apps**. It also enables testing via invite-only [release channels](https://wearables.developer.meta.com/docs/develop/dat/set-up-release-channels).
+
+### Verify glasses software version
+
+1. In the Meta AI app, go to the Devices tab (the glasses icon at the bottom of the app), and select your device.
+2. Tap the gear icon to open **Device settings**.
+3. Tap **General** > **About** > **Version**.
+4. You should have the minimum supported version or above installed on your glasses, as outlined [here](https://wearables.developer.meta.com/docs/develop/dat/version-dependencies/).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)